app/routes/_seo+/sitemap[.]xml.ts

```diff
@@ -0,0 +1,19 @@
+/*!
+ * Copyright © 2023 United States Government as represented by the
```

Comment:
I understand why this has to be here, but I don't know how I feel about merging this with the copyright disclaimer.

Do I need to put together a CLA for the first time in my life? 😬

Reply:
Oops, I forgot to remove that.